### PR TITLE
Build ubuntu cloud images in drone

### PR DESCRIPTION
**Pull request by gabriel-v**
The images come with the cluster docker image already pulled and running example configs.

### CODE REVIEW

File: ci/provision-vm.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -ex
```
Comment (mgax) [inline ci/provision-vm.sh]: You know, this whole file, it should really be called `examples/pwn.sh`, or, more politely, `examples/install.sh`.

File: cluster.py [hunk 2]
```diff
@@ -90,16 +91,17 @@ def consul_retry_join_section(servers):
     quoted = [f'"{ip}"' for ip in servers]
     return f'retry_join = [{", ".join(quoted)}]'
 
+
 class OPTIONS:
-    nomad_interface = config.get('nomad', 'interface', fallback=None) or detect_interface()
-    _nomad_meta = {key: config.get('nomad_meta', key) for key in config['nomad_meta']} if 'nomad_meta' in config else {}
-    nomad_meta = "\n".join(f'{key} = "{value}"' for key, value in _nomad_meta.items())
+    nomad_interface = config.get('nomad', 'interface', fallback=None) or detect_interface()  # noqa: E501
```
Comment (mgax) [inline cluster.py]: What with all them `noqa`? Can't we just disable rule 501 or something? At least for this one file?

File: ci/provision-vm.sh (hunk 1 above)
Comment (gabriel-v) [reply ci/provision-vm.sh]: It's unpacking `/opt/cluster.tar.gz` and injecting some other magic ci-related magic, so I wouldn't have this in the examples right now

File: cluster.py (hunk 2 above)
Comment (gabriel-v) [reply cluster.py]: I think the comments are a good reminder to refactor this horrible code.

**Pull request merged**